mealie/routes/validators/validators.py

```diff
@@ -0,0 +1,45 @@
+from fastapi import APIRouter, Depends
+from typing import Union
+from sqlalchemy.orm.session import Session
+
+from mealie.db.db_setup import generate_session
+from mealie.repos.all_repositories import get_repositories
+from mealie.schema.user import UserBase
+from mealie.schema.response import ErrorResponse, SuccessResponse
+from mealie.db.models.users import User
+from mealie.db.models.group import Group
+from mealie.db.models.recipe.recipe import RecipeModel
+
+
+router = APIRouter()
+
+
+def create_response(value, type: str):
+    if value is None:
+        return SuccessResponse.respond(message=f"No {type} exists")
+    return ErrorResponse.respond(message=f"{type.capitalize()} already exists")
```

Comment:
SuccessResponse and ErrorResponse are used for flash messages on the frontend. There's some middleware on the frontend that will check for the message property and display them to the user. 

Since the intention of these is to provide some real-time validation for text fields it would be better to use a dedicated model that returns a valid: true/false payload so we can handle the error messages on frontend on a case by case basis.

```python
from pydantic import BaseModel

class ValidationResponse(BaseModel):
    valid: bool = False


```